### PR TITLE
C18 WMDE EN Test 04

### PR DESCRIPTION
**Pull request by ghost**
[T211083](https://phabricator.wikimedia.org/T211083)

### CODE REVIEW

File: english/banner_ctrl.js [hunk 1]
```diff
@@ -37,7 +34,7 @@ const campaignProjection = new CampaignProjection(
 	CampaignParameters.donationProjection
 );
 const formatNumber = require( 'format-number' );
-const donorFormatter = formatNumber( { round: 0, integerSeparator: '.' } );
+const donorFormatter = formatNumber( { round: 0, integer1Separator: '.' } );
```
Comment (tzhelyazkova) [inline english/banner_ctrl.js]: looks like a typo? :thinking:

File: english/templates/application_of_funds.hbs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<div class="wlightbox-contents">
```
Comment (tzhelyazkova) [inline english/templates/application_of_funds.hbs]: Why is this file re-introduced?

**Review by tzhelyazkova — CHANGES_REQUESTED** [review]
It's not supa dupa clear from the ticket's AC but I think `The button labels use English language and localization (e. g. "€ 10").` might apply for both `var` and `ctrl`.

File: english/banner_ctrl.js (hunk 1 above)
Comment (ghost) [reply english/banner_ctrl.js]: 1

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**